client/config/environment.js

```diff
@@ -18,6 +17,8 @@ module.exports = function (environment) {
     NYC_ID_HOST,
     maintenanceTimes: getMaintenanceTimes(),
     showAlerts: getFeatureFlagShowAlerts(),
+    featureFlagShowSandboxWarning: getFeatureFlagShowSandboxWarning(),
```

Comment:
This is probably splitting hairs but do we consider this a "feature flag" _per say?_ @TangoYankee - thoughts?

Reply:
It's a feature and we need to toggle it with environment flags :nerd_face: